sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -5010,6 +5010,8 @@ end subroutine calchipsi
 !> -----|------------|---------
 !> 2024-11-15 | Jesse Meng | Initial
 !> 2025-04-23 | Jesse Meng | Bug fix zmid calculation in very thin layers
+!> 2025-06-12 | Jesse Meng | Bug fix RH converted from decimal to percent
+!> 2025-06-12 | Jesse Meng | SLR masking procedure modified
```

Comment:
@JesseMeng-NOAA Could you combine these comments onto one line?  
Bug fix RH converted from decimal to percent, SLR masking procedure modified
Thanks!

Reply:
Updated.